src/features/Console.ts

```diff
@@ -211,7 +211,8 @@ export class ConsoleFeature implements IFeature {
                     return;
                 }
 
-                if (vscode.window.activeTerminal.name !== "PowerShell Integrated Console") {
+                if (vscode.window.activeTerminal &&
```

Comment:
Would this flow be better?

```
if (!activeTerminal && existingPsExtTerminal) {
  // ShowSessionConsole
  return;
}
if (!activeTerminal || activeTerminal !== "PowerShell Integrated Console") {
  // Use runSelectedText
  return;
}
// ShowSessionConsole
```

Reply:
@Tyriar I'm guessing from your snippet that `existingPsExtTerminal` would be used to determine if the PS extensions terminal is the "active"/"visible" terminal?